services/notifications/src/simcore_service_notifications/services/postgres/__init__.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from collections.abc import AsyncIterator
+
+from fastapi import FastAPI
+from fastapi_lifespan_manager import State
+from servicelib.fastapi.db_asyncpg_engine import (
+    close_db_connection,
+    connect_to_db,
+)
+from servicelib.fastapi.postgres_lifespan import PostgresLifespanState
+from servicelib.logging_utils import log_context
+
+from ...core.settings import ApplicationSettings
+from ._health import PostgresHealth
+
+_logger = logging.getLogger(__name__)
+
+
+async def postgres_lifespan(app: FastAPI, state: State) -> AsyncIterator[State]:
+    app.state.engine = state[PostgresLifespanState.POSTGRES_ASYNC_ENGINE]
+
+    settings: ApplicationSettings = app.state.settings
+
+    app.state.postgress_liveness = PostgresHealth(app)
+
+    with log_context(_logger, logging.INFO, msg="connecting to postgres"):
+        await connect_to_db(app, settings.NOTIFICATIONS_POSTGRES)
```

Comment:
Please use the `servicelib.fastapi.postgres_lifespan` in conjunction with thisone.  Note that the former guarantees the connection/disconnection of the database so you do not need to do it here